docs/codeql/codeql-for-visual-studio-code/about-telemetry-in-codeql-for-visual-studio-code.rst

```diff
@@ -30,6 +30,7 @@ If you opt in, GitHub collects the following information related to the usage of
 - Randomly generated GUID that uniquely identifies a CodeQL extension installation. (Discarded before aggregation.)
 - IP address of the client sending the telemetry data. (Discarded before aggregation.)
 - Whether or not the ``codeQL.canary`` setting is enabled and set to ``true``.
+- Whether any :doc:`CodeQL extension settings <customizing-settings>` are configured.
```

Comment:
`codeQL.canary` is also a CodeQL extension setting, so from a technical perspective it's odd to mention them in two separate lines. But I don't know what's more understandable for the user. I thought I'd mention this, but I think what's written here already is a perfectly fine explanation.

Reply:
Yep, I wondered about that too! @jf205, any preference on whether we continue to call out the `canary` setting separately, or just include the general line about settings?